spec/models/authorization_request_spec.rb

```diff
@@ -33,6 +33,25 @@
     end
   end
 
+  describe '.viewable_by_users scope' do
+    subject { described_class.viewable_by_users }
+
+    let!(:authorization_request) do
+      create(:authorization_request, :with_tokens, api: 'particulier', status: 'archived')
+      create(:authorization_request, api: 'particulier', status: 'revoked')
+      create(:authorization_request, status: 'validated')
+    end
```

Comment:
très bizarre ce que tu fais ici, in fine c'est que le dernier qui est affecté à `authorization_request`, mais je suppose que tu voulais tester que l'ensemble était là ? si oui faut que tu fasses un array (et que tu renames le let).